Source/WebInspectorUI/UserInterface/Views/ScopeBar.css

```diff
@@ -43,7 +43,7 @@
     font-size: 11px;
     line-height: 11px;
     color: var(--scope-bar-text-color);
-    text-align: center;
+    text-align: right;
```

Comment:
this will likely have much more far reaching consequences than just in the Audit Tab since `WI.ScopeBar` is used all over the place

i think it'd be better to do something similar to the other variables below (e.g. `--scope-bar-text-align: var(--scope-bar-text-align-override, var(--scope-bar-text-align-default))`) so that we can have greater visibility in the future as to where this value is overridden from the default

Reply:
the default should remain as it is today (i.e. `--scope-bar-text-align-default: center;`) and then you'd override it where needed, e.g.:

```css
.content-view.audit-test-group > header > nav > .scope-bar {
    ...

    --scope-bar-text-align-override: end;
}
```

Reply:
Where else are the scope bars in use in the web inspector?